### DISCUSSION TITLE
Unit tests in Titiler are failing because of the httpx version 0.25.0.

### DISCUSSION THREAD

**Discussion by raster-blaster** [developmentseed/titiler#696] [Q&A]
The 'httpx' library was upgraded to version 0.25.0 yesterday, and since then, unit tests in Titiler have been failing. The issue is resolved by downgrading 'httpx' to version 0.24.1.

In the 'src/titiler/core/pyproject.toml' file, the test dependencies for 'httpx' do not specify a version number. Could we please add the version number for 'httpx' in this file?

**Comment by vincentsarago** [comment]
I'm off this week but I could review any PR 🙌
